install_config/topics/ocp_osp_prerequisites.adoc

```diff
@@ -0,0 +1,35 @@
+=== {product-title} Prerequisites
+
+A successful deployment of {product-title} requires many prerequisites. This consists of
+a set of infrastructure and host configuration steps prior to the actual
+installation of {product-title} using Ansible. In the following subsequent sections, details
+regarding the prerequisites and configuration changes required for an {product-title} on a
+OpenStack environment are discussed in detail.
+
+NOTE: All the OpenStack CLI commands in this reference environment are executed
+using the CLI `openstack` commands within the OpenStack director node. If using a 
+workstation or laptop to execute these commands instead of the OpenStack
+director node, please ensure to install the following packages found
+within the specified repositories.
+
+Example:
+
+Enable the rhel-7-server-openstack-{rhosp_version}-rpms and Ansible repositories
+
+----
+$ sudo subscription-manager repos \
+--enable rhel-7-server-openstack-13-rpms \
+--enable=rhel-7-server-ansible-2.5-rpms
+$ sudo yum install -y python-openstackclient python2-heatclient python2-octaviaclient ansible
+----
+
+Verify the packages are of at least the following versions (use `rpm -q <package_name>`):
+
+* `python-openstackclient` - `3.14.1.-1`
```

Comment:
# rpm -q python2-openstackclient
python2-openstackclient-3.14.2-1.el7ost.noarch

So, change 'python-openstackclient' to 'python2-openstackclient'